docs/conda-env.yml

```diff
@@ -0,0 +1,17 @@
+name: qtconsole_docs
+channels:
+- asmeurer
+dependencies:
+- python=3.4.3=2
+- nbformat
+- jupyter_client
+- ipykernel
+- numpydoc
+- qt
+- pexpect
+- pygments
+- pyzmq
+- pyqt
+- pip:
+    - recommonmark
```

Comment:
and `readthedocs-sphinx-ext` neither, but readthedocs crash if we dont install them. Same for `numpydoc`.